fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -850,6 +850,8 @@ private static void verifyTimeMapHeaders(final CloseableHttpResponse response, f
         checkForLinkHeader(response, uri + "/" + FCR_VERSIONS, "timemap");
         checkForLinkHeader(response, VERSIONING_TIMEMAP_TYPE, "type");
         checkForLinkHeader(response, ldpcvUri + "/" + FCR_ACL, "acl");
+        assertFalse(response.getFirstHeader("Allow").getValue().contains("DELETE"));
+        assertTrue(response.getFirstHeader("Allow").getValue().contains("POST,HEAD,GET"));
```

Comment:
Would you mind checking these individually... it seems a little fragile to expect the exact string of `POST,HEAD,GET`.
Thanks!

Reply:
I can, I thought it'd be OK as the options are held as a fixed string, so the string wouldn't change

Reply:
If you could make the change, it would be helpful.